conda-forge.yml

```diff
@@ -1,9 +1,13 @@
-bot: {automerge: true}
+bot:
+  automerge: true
 compiler_stack: comp7
 conda_forge_output_validation: true
 max_py_ver: '37'
 max_r_ver: '35'
```

Comment:
Some keys here are really outdated, and we should also specify the default testing strategy.
```suggestion
conda_forge_output_validation: true
test: native_and_emulated
```